go/consensus/tendermint/tendermint.go

```diff
@@ -710,12 +710,10 @@ func (t *tendermintService) GetTransactions(ctx context.Context, height int64) (
 }
 
 func (t *tendermintService) GetStatus(ctx context.Context) (*consensusAPI.Status, error) {
-	genDoc, err := t.GetGenesisDocument(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	genBlk, err := t.GetBlock(ctx, genDoc.Height)
+	// Genesis block is hardcoded as block 1, since tendermint doesn't have
```

Comment:
Note that this is only true until Tendermint starts supporting restoring from a non-zero height. Maybe mention this and reference https://github.com/tendermint/tendermint/issues/2543.